siwe_auth/views.py

```diff
@@ -20,7 +23,17 @@
 @ratelimit(key='ip', rate='5/m')
 @require_http_methods(["POST"])
 def login(request):
-    wallet = authenticate(request)
+    body = json.loads(request.body)
+    auth_kwargs = {
+        "siwe_message": SiweMessage(
+            message={
+                re.sub(r"(?<!^)(?=[A-Z])", "_", k).lower(): v
```

Comment:
Minor style thing, but I think I'd take this re function and wrap it in a function with a descriptive name. Totally up to your preference though

Reply:
Good idea! Done!